pspamm/matmul.py

```diff
@@ -233,6 +235,8 @@ def make_nk_unroll(self):
             if self.alpha == 1.0 and self.beta != 0.0:
                 asm.add(self.generator.move_register_block(self.C, C_ptr, Coords(), regs, self.v_size, self.additional_regs, None, False))
                 if self.beta != 1.0:
+                    if self.use_bcst:
+                        asm.add(bcst(pspamm.architecture.operands.r(4), self.beta_reg[1], "Broadcast beta"))
```

Comment:
Maybe turn `pspamm.architecture.operands.r(4)` into a parameter, like the `self.beta_reg`s?